### ISSUE TITLE
short range 

### ISSUE THREAD

**Issue by mikerc2017** [fishpepper/OpenSky#20]
hello Simon
About VERSIONS cc2510  ( tinyfish fc or usky  )
i get short range < 30m even  <20m  when fly place many obstacles . Do you plan  support version full range ?

**Comment by mikerc2017** [comment]
in target usky , i saw line 

// LNA control pin
// #define RF_LNA_PORT                 P0
// #define RF_LNA_PIN                  0 
// #define RF_LNA_ON_LEVEL             1
// PA control pin
// #define RF_PA_PORT                  P0
// #define RF_PA_PIN                   1
// #define RF_PA_ON_LEVEL              1
// enable high gain?
// #define RF_HIGH_GAIN_MODE_ENABLED
// gain control pin
// #define RF_HIGH_GAIN_MODE_PORT      P1
// #define RF_HIGH_GAIN_MODE_PIN       0
// #define RF_HIGH_GAIN_MODE_ON_LEVEL  1

It ready work ?

**Comment by squarecode** [comment]
Unfortunatly this feature requires some additional hardware. The uSKY was in my opinion just a short range indoor receiver.
Correct me if I'm wrong...

**Comment by mikerc2017** [comment]
yes , use add a PA/LNA for better range.   i saw usky not enough indoor many floor/room . Taranis warring RSSI   noise..

**Comment by MJ666** [comment]
Actually the AlienFlightNG team is working on an full range receiver for the AFRX OpenSky target. This will have LNA/PA and Diversity support. Stay tuned. The RX will be release here when we finished testing: www.//alienflightng.com

**Comment by mikerc2017** [comment]
Ohh .i miss this infor . Thanks MJ666.  hope it will soon, i will follow Alienflight.

**Comment by mikerc2017** [comment]
hello Mj666
I saw you modify code for AFRX , IT COMPLETED ? can you  send me file hex to test ?  i want to test soon . 
thanks

**Comment by MJ666** [comment]
You also need the related hardware for it? We are just starting to build an test our prototypes. What do you want to do with the hex file?

**Comment by mikerc2017** [comment]
yes. i need a firmware. I can build hardware with PA/LNA  now to test . Will be great if it work.
if can , please send it for me through email  . thanks

**Comment by MJ666** [comment]
If you provide me with your email address I can send you the AFFX hex files containing the latest changes. Please be aware we are actual going to test this changes with our AFNG RX prototypes. Actually the hardware for the prototypes is build. After the tests are successfully finalized you will find the design files for the receiver at www.alienflightng.com.

**Comment by mikerc2017** [comment]
i sent you a email . Check inbox email .
And i still follow information from alienflight

**Comment by mikerc2017** [comment]
My mail jamedavid300688@gmail.com.
And i still follow new information from alienflight

**Comment by MJ666** [comment]
Send firmware files by email.

**Comment by ghost** [comment]
@MJ666 Any updates on that AlienFlightNG project? Seems great to me =)

**Comment by Pedro147** [comment]
I am sure that mj666 will reply to you but in the meantime have a look on their website here https://www.alienflightng.com/

**Comment by MJ666** [comment]
We don't had much success with our full range prototype RX. All variants we build and tested had still range problems. Likely it was related to our RF design of the PCB's. We finally stopped the project. We had an FC design in the queue with using an ready made commercial CC2500 module but also get stuck here since many FC are already out there and a new AlienFlightNG design will not have much advantages. There are only limited number people who are able to DYS building this controllers and we don't had plans to go commercial.

**Comment by Pedro147** [comment]
It's a pity Michael but the AlienFlightNG designs were a great thing at the time. RX design is quite complicated from all accounts so no shame in the fact that they didn't perform as expected. I and many others thank you, Gary and Leon for all the work you did after the Lance AlienFlight/ AlienWii era

**Comment by ghost** [comment]
@MJ666 That's sad. Anyway, thanks for all the efforts
